src/model/version_list.rs

```diff
@@ -3,10 +3,35 @@ pub struct VersionList {}
 
 impl VersionList {
     pub fn latest_version(versions: Vec<Version>) -> Option<Version> {
-        let mut sorted_versions = versions.to_vec();
-        sorted_versions.sort_by(|a, b| b.uploaded_at.cmp(&a.uploaded_at));
+        let sorted_versions = VersionList::sort_versions(versions);
         sorted_versions.first().map(|v| v.clone())
     }
+
+    pub fn latest_version_of_distribution_group(versions: Vec<Version>, distribution_group: String) -> Option<Version> {
```

Comment:
If you check a bit further down in the file, you see that you can write unit tests for this. Could you add a test?